components/stratum/include/stratum.h

```diff
@@ -0,0 +1,69 @@
+#ifndef STRATUM_COMPONENT_H
+#define STRATUM_COMPONENT_H
+
+#include "cJSON.h"
+#include <stdint.h>
+
+#define REQUEST_BUFFER_SIZE 1024
+
+// Predefined message IDs
+#define STRATUM_CONFIGURE_MESSAGE_ID 1
+#define STRATUM_SUBSCRIBE_MESSAGE_ID 2
+#define STRATUM_AUTHORIZE_MESSAGE_ID 3
+#define STRATUM_SUGGEST_DIFFICULTY_MESSAGE_ID 4
```

Comment:
TODO: Add a note why these should never NOT be these IDs  - no need to be dynamic - we don't know what it looks like when things restart w/ the pool but we're using different IDs.